src/Rings/mpoly-ideals.jl

```diff
@@ -2068,9 +2069,9 @@ end
 @doc raw"""
     grassmann_pluecker_ideal([ring::MPolyRing,] subspace_dimension::Int, ambient_dimension::Int)
 
-Given a ring, an ambient dimension and a subspace dimension return the ideal in the given ring
+Given a (possibly graded) ring, an ambient dimension and a subspace dimension return the ideal in the given ring
```

Comment:
One more observation:
You are actually changing the ring in the code corresponding to this docstring, but the docstring says 'given ring'. It is no longer an MPolyRing, but an MPolyDecRing after the use of `grade. Is this on purpose or by accident?